netpyne_ui/experiments.py

```diff
@@ -220,11 +220,14 @@ def _create_trials(experiment: model.Experiment) -> List[model.Trial]:
 
     for param in params:
         if param.type == "range":
-            param.values = list(np.arange(param.min, param.max, param.step))
+            if param.step > 0:
```

Comment:
This is simple atm but it would be nicer to have the validator as a separate (and testable) function/object

Reply:
yes agree